velox/functions/remote/client/Remote.cpp

```diff
@@ -62,96 +85,25 @@ class RemoteFunction : public exec::VectorFunction {
       exec::EvalCtx& context,
       VectorPtr& result) const override {
     try {
-      applyRemote(rows, args, outputType, context, result);
+      if (remoteClient_) {
+        remoteClient_->applyRemote(rows, args, outputType, context, result);
+      } else {
+        VELOX_FAIL("Remote client not initialized.");
```

Comment:
this could also be:

```
VELOX_CHECK(remoteClient_, "Remote client not initialized.");
remoteClient_->applyRemote(rows, args, outputType, context,
```